### DISCUSSION TITLE
Pixelated model on Firefox

### DISCUSSION THREAD

**Discussion by wawa2016** [google/model-viewer#1971] [Q&A] [answered]
I have some problems with my model showing pixelated/ blurred on Firefox. It seems to work well on Chrome.
Could you suggest what the problem is? 
I used minimumRenderScale="1" but it does not seem to fix it.

**Comment by elalish** [comment, marked as answer]
I think you missed the documentation about how to set static properties: https://modelviewer.dev/examples/loading/#dracoSupport

try `customElements.get('model-viewer').minimumRenderScale = 1;`
